lib/podsecuritypolicy.libsonnet

```diff
@@ -0,0 +1,21 @@
+/**
+ * \file Helper to create PodSecurityPolicy objects.
+ */
+
+local com = import 'lib/commodore.libjsonnet';
+local kap = import 'lib/kapitan.libjsonnet';
+local kube = import 'lib/kube.libjsonnet';
+local inv = kap.inventory();
+local params = inv.parameters.podsecuritypolicy;
+
+local podSecurityPolicy(name) = kube._Object('policy/v1beta1', 'PodSecurityPolicy', name) {
```

Comment:
Please add the same comments as here for example: https://github.com/projectsyn/component-storageclass/blob/master/lib/storageclass.libsonnet#L12

Reply:
done